### PR TITLE
make init script order configurable

### PR DESCRIPTION
**Pull request by ctramnitz**
have chef-client start before another service, first implementation is rhel<7-only (suse doesn't seem to use chkconfig, debian has different chkconfig variable in template, but unclear if being used, system needs before/after handling instead of numeric value)

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @ctramnitz

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

File: attributes/default.rb [hunk 1]
```diff
@@ -124,6 +124,8 @@
   default['chef_client']['run_path']    = '/var/run/chef'
   default['chef_client']['cache_path']  = '/var/cache/chef'
   default['chef_client']['backup_path'] = '/var/lib/chef'
+  default['chef_client']['chkconfig']['start_order'] = 98
+  default['chef_client']['chkconfig']['start_order'] = 02
```
Comment (cheeseplus) [inline attributes/default.rb]: Probably meant stop_order for one of these

Comment (ctramnitz) [inline attributes/default.rb]: yes, of course it was meant to be like this

**Comment by ctr49** [comment]
@cheeseplus could you please review again? requested changes have been made.

Thanks!

**Comment by tas50** [comment]
Sorry for the delay. This will ship with the upcoming major release.

**Pull request merged**